src/introduction/ccc.md

```diff
@@ -6,17 +6,15 @@ ccc <arch> [...]
 
 Supported architectures:
 	aarch64         - conventional AArch64
-	morello-hybrid  - AArch64 Morello supporting CHERI
 	morello-purecap - AArch64 Morello pure-capability
 	riscv64         - conventional RISC-V 64-bit
-	riscv64-hybrid  - RISC-V 64-bit supporting CHERI
 	riscv64-purecap - RISC-V 64-bit pure-capability
```

Comment:
This is taken verbatim from ccc itself, so probably best to include it and just make sure we bring back the old text about hybrid being unused in these exercises with suitable generalisation? Otherwise people might look at ccc, notice morello-hybrid and be confused about which of the two morellos they're supposed to be using, and what the difference is. At least the observant ones; I'm sure many won't even notice.

Reply:
Should we perhaps mark them as [unused] or some sort of annotation in the doc?